custom/icds_reports/utils/aggregation_helpers/child_health_monthly.py

```diff
@@ -280,7 +280,7 @@ def _state_aggregation_query(self, state_id):
                       date_trunc('MONTH', child_tasks.due_list_date_7g_vit_a_9) = %(start_date)s
                   THEN 1 ELSE NULL END
             """),
-            ("mother_phone_number", "person_cases.phone_number"),
+            ("mother_phone_number", "person_cases.contact_phone_number"),
```

Comment:
This is really confusing and I definitely would not expect you to know this but the `person_cases` join actually joins the person case of the child, not the mother, despite the fact that it says `mother_id`. That column is actually the id of the person case corresponding to the child health case not the mother. I think you need to add a column to the child health ucr, either a real reference to the mother case id (using the `mother_id` expression, which is correct) and then doing a second join on the person table or just directly pulling in the mother number. Otherwise this looks good

Reply:
It would be nice to add a comment to that UCR as well for both of those columns to try to clear up confusion